awx-operator.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: awx-operator
+  version: 2.19.1
+  epoch: 0
+  description: An Ansible AWX operator for Kubernetes built with Operator SDK and Ansible.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ansible-operator
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - py3-ansible-core
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/ansible/awx-operator
+      tag: ${{package.version}}
+      expected-commit: dd37ebd440edf953d822f2c134833a44a8e77532
+
+  - runs: |
+      mkdir -p ${{targets.contextdir}}/opt/ansible
+      cp requirements.yml ${{targets.contextdir}}/opt/ansible/requirements.yml
+      HOME=${{targets.contextdir}}/opt/ansible ansible-galaxy collection install -r ${{targets.contextdir}}/opt/ansible/requirements.yml
+      cp watches.yaml ${{targets.contextdir}}/opt/ansible/watches.yaml
+      cp -r roles ${{targets.contextdir}}/opt/ansible/roles
+      cp -r playbooks ${{targets.contextdir}}/opt/ansible/playbooks
+
+update:
+  enabled: true
+  github:
+    identifier: ansible/awx-operator
+
+test:
+  pipeline:
+    - name: Check Ansible files
+      runs: |
+        stat /opt/ansible/requirements.yml
+        stat /opt/ansible/watches.yaml
+        stat /opt/ansible/roles
+        stat /opt/ansible/playbooks/awx.yml
```

Comment:
`test -f` (test if a file) and `test -d` (directory) would be a better and more explicit test.

But you're explicitly copying most things so I'd only leave the test for `playbooks/awx.yaml` ass you're copying an entire directory. You could do the same for one of the files in `roles` directory.

Reply:
Good point. As discussed in general maintaining specific file checks over time can be cumbersome. In this case I'd just test `watches.yaml` playbook file as it's the one expected by default by the `run` command.